src/IO/Messenger.php

```diff
@@ -468,7 +468,28 @@ private function decorateTemplateSource(Template\TemplateSource $templateSource)
             return $name;
         }
 
-        return sprintf('%s <fg=gray>(%s)</>', $description, $name);
+        return sprintf(
+            '%s <fg=gray>(%s)</>',
+            $description,
+            $this->disclaimAbandonedPackagesInTemplateSource($package),
+        );
+    }
+
+    private function disclaimAbandonedPackagesInTemplateSource(Package\CompletePackageInterface $package): string
+    {
+        if (!$package->isAbandoned()) {
+            return $package->getPrettyName();
+        }
+
+        return sprintf(
+            '%s %s%s',
+            $package->getPrettyName(),
+            'is abandoned.',
+            match ($package->getReplacementPackage()) {
+                null => ' No replacement was suggested.',
+                default => ' Use '.$package->getReplacementPackage().' instead.',
+            },
+        );
```

Comment:
See my comment in the PR. I'd suggest the following, if we choose the other output:

```suggestion
        $decoratedTemplateSource = sprintf('%s <fg=gray>(%s)</>', $description, $name);

        if (!$package->isAbandoned()) {
            return $decoratedTemplateSource;
        }

        return sprintf(
            '%s <warning>Abandoned! %s</warning>',
            $decoratedTemplateSource,
            match ($package->getReplacementPackage()) {
                null => 'No replacement was suggested.',
                default => 'Use '.$package->getReplacementPackage().' instead.',
            },
        );
```